packages/backend/src/workers/provider/LlamaCppPython.ts

```diff
@@ -56,6 +58,18 @@ export class LlamaCppPython extends InferenceProvider {
 
     const envs: string[] = [`MODEL_PATH=/models/${modelInfo.file.file}`, 'HOST=0.0.0.0', 'PORT=8000'];
     envs.push(...getModelPropertiesForEnvironment(modelInfo));
+    if (isMacArm()) {
+      envs.push('GPU_LAYERS=99');
```

Comment:
Why did you add 99 to layers? I recall that by default it uses -1 which is all it can. Shouldn't it be a value given by the user?